lib/daru/io/io.rb

```diff
@@ -275,6 +275,17 @@ def html_table_to_dataframe(table)
           order: table[:order],
           name: table[:name]
       end
+
+      CSV::Converters[:boolean_converter] = lambda do |f, _|
```

Comment:
The converters option of the CSV parser accepts either the name (symbol) of a converter in the `CSV::Converters` hash OR a lamba.  So, you could probably build a `Daru::CSV::Converters` hash for Daru-specific converters, and then have `DataFrame.from_csv` accept converter names that first look in `CSV::Converters` and then in `Daru::CSV::Converters`.  That way a user could override the Daru ones by either adding to `CSV::Converters` or supplying their own lambda.